padle/view/padle_scroll_view.py

```diff
@@ -0,0 +1,104 @@
+from math import ceil
+from typing import Collection, TYPE_CHECKING
+
+from discordmenu.embed.components import EmbedMain, EmbedField
+from discordmenu.embed.base import Box
+from discordmenu.embed.view import EmbedView
+from tsutils.menu.components.footers import embed_footer_with_state
+from tsutils.menu.components.config import UserConfig
+from tsutils.menu.view.view_state_base import ViewStateBase
+from tsutils.query_settings.query_settings import QuerySettings
+from padle.monsterdiff import MonsterDiff
+
+from discordmenu.embed.components import EmbedFooter
+
+from tsutils.tsubaki.links import CLOUDFRONT_URL
+
+TSUBAKI_FLOWER_ICON_URL = CLOUDFRONT_URL + '/tsubaki/tsubakiflower.png'
```

Comment:
This is in `tsubaki.menu.components.footers` as `TSUBAKI_FLOWER_ICON_URL`

Reply:
also there is no need to import it period because embed_footer_with_state or something completely encapsulates it

Reply:
since the implementation of the embed isn't menu2, there isn't any state - I just thought the flower looked nice next to the text lol.

Reply:
are you sure? isn't this the one I opened a PR for?

```python
class PADleScrollView:
    VIEW_TYPE = 'PADleScroll'

    @staticmethod
    def embed(state: PADleScrollViewState):
        fields = state.get_cur_fields()
        return EmbedView(
            EmbedMain(
                title=f"**PADle #{state.current_day}**",
            ),
            embed_footer=embed_footer_with_state(state, text=state.get_pages_footer()),
            embed_fields=fields)
```

Reply:
wait sorry thought the comment referred to a different part in code